clouddriver-artifacts/src/main/java/com/netflix/spinnaker/config/ArtifactConfiguration.java

```diff
@@ -25,13 +28,22 @@
 import org.springframework.stereotype.Component;
 
 @Configuration
-@EnableConfigurationProperties
 @EnableScheduling
 @Component
+@EnableConfigurationProperties(ArtifactProviderProperties.class)
 @ComponentScan("com.netflix.spinnaker.clouddriver.artifacts")
+@Slf4j
 public class ArtifactConfiguration {
+
   @Bean
-  OkHttpClient okHttpClient() {
-    return new OkHttpClient();
```

Comment:
Base client is already configured here [kork-web]( https://github.com/spinnaker/kork/blob/master/kork-web/src/main/java/com/netflix/spinnaker/config/okhttp3/RawOkHttpClientConfiguration.java#L45) and we should be able to inject this and customize further for timeout settings etc using the `newBuilder` api.